ament_cmake_python/ament_cmake_python-extras.cmake

```diff
@@ -40,7 +40,7 @@ macro(_ament_cmake_python_register_environment_hook)
   endif()
 endmacro()
 
-macro(_ament_cmake_python_get_python_install_dir)
+macro(ament_cmake_python_get_python_install_dir)
```

Comment:
@MaximilienNaveau now that this is public API, a documentation block would be nice.

Reply:
I recommend making this a `function()` too with an output variable to avoid other variables it sets (_python_code, _output, _result) from being visible to the code that calls this. I also recommend shortening the name.

This

`_ament_cmake_python_get_python_install_dir()`

would become this:

`ament_get_python_install_dir(PYTHON_INSTALL_DIR)`

Reply:
I do agree I will do the change whenever I have some time!

Reply:
@MaximilienNaveau friendly ping

Reply:
Sorry I have been under water lately I will try to do this next week!